locations/k9.yml

```diff
@@ -34,48 +37,71 @@ ipv6_prefix: '2001:bf7:830:8d00::/56'
 # - 10.31.9.224/28 - BBB-Mesh
 # - 10.31.9.240/28 - Internal Mesh
 
-
 # 10.31.99.0/24 / can be proably dismantled
 
-
 networks:
+  # MESH - Sama
   - vid: 10
     role: mesh
     name: mesh_sama
     prefix: 10.31.9.224/32
-    ipv6_subprefix: -1
-    mesh_metric: 512
+    ipv6_subprefix: -10
     ptp: true
 
+  # MESH - Zwingli
   - vid: 11
     role: mesh
     name: mesh_zwingli
     prefix: 10.31.9.225/32
-    ipv6_subprefix: -2
+    ipv6_subprefix: -11
     mesh_metric: 1024
-    mesh_metric_lqm: ['default 0.3']
-    # Ignore Uplink two Hops away / requires 0.3 LQM
+    mesh_metric_lqm: ['default 0.6']
```

Comment:
It's not about the uplink routing, it's about the routing between sama<->zwingli in case of 60 GHz failing. We don't want it to be via k9 in this case, we want it to prefer the route via gruni with modern hardware for both links. I could have adjusted it at zwingli instead but I think it's fine to leave it in here for now.